appendix_meta_rules.md

```diff
@@ -104,6 +104,49 @@ Example: A valid GPO script that triggers multiple Sigma rules.
 
 All rules in a file, basic event rules as well as correlations, might contain an additional attribute called "generate".
 If it is set to true, the rule will generate a query, even if it is referred to by other correlations. Otherwise by default no "standalone" query would be generated for this rule.
```

Comment:
```suggestion
The rules in a multi-document YAML that build a correlation rule are not producing single, independent queries. They are used as a tool to define more complex constructs out of basic Sigma detections. Therefore only the outermost correlation rule may define meta information such as status, level, date or anything else.
```

It's pretty much under defined like this but the spirit of correlation rules should be more clear like that.

A question I asked myself is, if we even need to define the use case "subquery should generate it's own query" as you can just duplicate it as it's own rule. At least that's what I would suggest.